ui-admin/src/test-utils/mocking-utils.tsx

```diff
@@ -196,16 +198,40 @@ export const mockStudyEnvironmentConfig = (): StudyEnvironmentConfig => {
   }
 }
 
-/** returns a simple studyEnvContext object for use/extension in tests */
-export const mockStudyEnvContext: () => StudyEnvContextT = () => {
-  const sandboxEnv: StudyEnvironment = {
-    environmentName: 'sandbox',
+export const mockStudy = (): Study => {
```

Comment:
these all were being imported from ui-participant - just copied the same functions here.